src/schemas/lendingRecordSchema.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from "apollo-server";
+
+export const lendingRecordTypeDefs = gql`
+  type LendingRecord {
+    id: String!
+    customer: Customer!
+    lentDate: String!
+    returnDate: String
+    dueDate: String!
```

Comment:
Let's define Date as a new Scalar type. 

```
dueDate: Date!
```

Reply:
After a small fight with graphQL and JSON objects I did it